doc/build-osx.md

```diff
@@ -223,10 +223,13 @@ After configuration, you are ready to compile.
 Run the following in your terminal to compile Bitcoin Core:
 
 ``` bash
-cmake --build build     # Use "-j N" here for N parallel jobs.
-ctest --test-dir build  # Use "-j N" for N parallel tests. Some tests are disabled if Python 3 is not available.
+cmake --build build -j$(nproc)
```

Comment:
Could also use `-G ninja`, the ninja build system uses all available CPU threads by default 😎 
(not sure we'd want to recommend it as default tho)